app/Http/Controllers/UsersController.php

```diff
@@ -131,6 +131,13 @@ public function show($id)
             ])
         );
 
+        if ($userArray['data']['page']['data']['html'] === '' &&
+            (Auth::check() && Auth::user()->user_id !== $user->user_id)) {
+            if (($index = array_search('me', $userArray['data']['profileOrder'], true)) !== false) {
+                array_splice($userArray['data']['profileOrder'], $index, 1);
+            }
+        }
```

Comment:
Uh yeah, you shouldn't be editing props... but I'm pretty sure @nanaya meant for you to implement it differently, not for it to be done at a PHP level :P